source/MapDetailPanel.cpp

```diff
@@ -122,13 +123,33 @@ void MapDetailPanel::Draw()
 
 
 
+bool MapDetailPanel::Scroll(double dx, double dy)
+{
+	Point point = UI::GetMouse();
+	if(maxScroll && point.X() < Screen::Left() + 160 && point.Y() > Screen::Top() + 90
```

Comment:
The values 160 and 90 look like magic numbers that are likely also used in other parts of the code. Can we make such magic numbers named const static variables? (In an appropriate location. maybe in the anonymous namespace?)

Reply:
Not to mention 130.

If these magic values are driven by the current pieces of UI being used, they belong in an interfaces.txt specification. You can specify the relative bounds, text positions, pointer positions, static labels, etc. and query them from the code.

Reply:
Okay I'll learn how to use the interface.txt for these new values.
Often it comes from image sizes and conventions so it should be easily modifiable by plugins and such.
Those magic numbers where already there when I arrived but may as well add code in a proper way indeed! Also interface.txt can't be that hard to use, I hope

Reply:
Okay It's done, but I can only hope I used it the right way